### DISCUSSION TITLE
i need help with getting the dank memer grinder to work

### DISCUSSION THREAD

**Discussion by RAINBOWgamesYT** [BridgeSenseMC/Dank-Memer-Grinder#130] [Q&A] [answered]
when i try and do the python one in my console i get to this step pip install -r .\requirements.txt
then when i do the command it says this 
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '.requirements.txt'
im not sure if i did something wrong or not can someone pls help me

**Comment by red78massive1573** [comment, marked as answer]
have you run `git clone https://github.com/BridgeSenseDev/Dank-Memer-Grinder.git`?
have you run `cd Dank-Memer-Grinder`?
it showed the error because it can't find the requirements file.
